edit/completion/completion_mode.go

```diff
@@ -389,7 +389,6 @@ func (c *completion) ListRender(width, maxHeight int) *ui.Buffer {
 			col.WriteSpaces(remainedWidth, styleForCompletion.String())
 		}
 		bb.ExtendRight(col.Buffer(), 0)
-		remainedWidth = 0
```

Comment:
This code was here to maintain that `remainedWidth` always reflect the width that remains, so that if more code is added below, `remainedWidth` would have the correct value.

Please add a comment there instead of removing this assignment.